en/guide/installation.md

```diff
@@ -43,11 +45,12 @@ $ mkdir <project-name>
 $ cd <project-name>
 ```
 
-*Info: replace project-name by the name of the project.*
+<p class="Alert Alert--nuxt-green"><b>Info:</b> replace <code>&lt;project-name&gt;</nom-du-projet></code> by the name of the project.</p>
```

Comment:
To kept consistency with others « Info : » box (e.g. in directory-structure.md for design, e.g. routing.md for « Warning/Info » and not « WARNING/INFO »)